apps/aecore/src/aec_headers.erl

```diff
@@ -357,7 +394,9 @@ serialize_for_client(#key_header{} = Header, PrevBlockType) ->
           <<"beneficiary">>   => aehttp_api_encoder:encode(account_pubkey, Header#key_header.beneficiary),
           <<"target">>        => Header#key_header.target,
           <<"time">>          => Header#key_header.time,
-          <<"version">>       => Header#key_header.version},
+          <<"version">>       => Header#key_header.version,
+          <<"info">>          => aehttp_api_encoder:encode(contract_bytearray, Header#key_header.info)
```

Comment:
This means `cb_` prefix?

Reply:
I see yes:
https://github.com/aeternity/protocol/pull/315/files#diff-029785cf793779e9ae524bd4d3d3b548R37